doc/releases.md

```diff
@@ -154,6 +154,11 @@ were first added in this version. The relevant commits should already include
 `sed -i "s/REPLACEME/$VERSION/g" doc/api/*.md` or
 `perl -pi -e "s/REPLACEME/$VERSION/g" doc/api/*.md`.
 
+If this release includes any new deprecations it is necessary to ensure that
+those are assigned a proper static deprecation code. These are listed in the
+docs (see `doc/api/deprecations.md`) and in the source as `DEP00XX`. The code
+must be assigned a number (e.g. `DEP0012`).
```

Comment:
Yes, they should be made before landing, otherwise we could end up with nightlies that contain invalid/duplicate deprecation codes if we wait until the next release.